test/docker/run_docker.sh

```diff
@@ -5,7 +5,7 @@ set -e
 # reset test harness
 rm -rf test-harness
 rm -rf test/features
-git clone --single-branch --branch feature/box-storage https://github.com/algorand/algorand-sdk-testing.git test-harness
+git clone --single-branch --branch use_latest https://github.com/algorand/algorand-sdk-testing.git test-harness
```

Comment:
Reviewer note:  The algorand-sdk-testing branch change can be reverted once https://github.com/algorand/algorand-sdk-testing/pull/204 is merged.